examples/rust/descriptors/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use bdk_wallet::bitcoin::bip32::Xpriv;
+use bdk_wallet::bitcoin::Network;
+use bdk_wallet::KeychainKind;
+use bdk_wallet::template::{Bip86, DescriptorTemplate};
+use bdk_wallet::keys::bip39::{Language, Mnemonic, WordCount};
+use bdk_wallet::keys::{GeneratedKey, GeneratableKey};
+use bdk_wallet::miniscript::Tap;
+
+fn main() -> () {
+    let mnemonic: GeneratedKey<_, Tap> =
+    Mnemonic::generate((WordCount::Words12, Language::English))
```

Comment:
We are changing the example from working with raw randomness to direct creation of mnemonics. Mnemonics are not directly related to descriptors, and belong in a separate example that addresses the specifics of interop between mnemonics/recovery phrases and descriptors.

Reply:
mmm ok